### DISCUSSION TITLE
about food effect

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1025] [Questions & Problems] [answered]
hello team,
![image](https://user-images.githubusercontent.com/92715703/142590228-4fa3897e-ffbc-4073-b640-8a6f71348cbe.png)
when I use the default,my figure like this
![image](https://user-images.githubusercontent.com/92715703/142590032-6a7da8b0-d5a6-4795-b5c0-bcdcafc55862.png)

when I change it ,
![image](https://user-images.githubusercontent.com/92715703/142590402-cc68ac3d-5cf0-4ace-b2f6-527547afe782.png)
the figure comes to 
![image](https://user-images.githubusercontent.com/92715703/142590315-e68cdc23-1900-467d-9bfc-f8bb43465bfd.png)

so I want to make sure what the tablet time delay factor means?
and can it use to explain the food effect?(like the food delays drug  release?)

**Comment by AndreDlm** [comment, marked as answer]
Dear @Carrie-nothing,

the `Tablet time delay factor` delays the transfer of the solid dosage form along the GI tract relative to water (but the dissolution rate itself is not directly affected). You can find the parameter when exporting your simulation to MoBi and navigating to the `Events` Building Block:

![image](https://user-images.githubusercontent.com/26541603/142603878-ccdc3e53-5d41-425d-bbfe-085b9bd413a1.png)

You can also visualize the tablet location in the GI tract by selecting `Plot parameter` in each of the `Tablet_Location_{GI-segment}` parameters and plot it in the simulation (e.g. `Tablet_Location_cae` will be 1 when the tablet is located in the cecum and 0 otherwise). 
Note that this only works when you have have set `IsSuspension` = `No`. In the default case (where `IsSuspension` is activated), the `suspension fraction` will be changed.

Anyway, in your example the transfer of the tablet along the GI tract is slowed resulting a greater Tmax. 

When simulating a food effect (via an added food intake in the `Events` building block in PK-Sim), the gastric emptying time is automatically increased, resulting in a slower transfer of *all* drug amount from stomach to duodenum. 
In contrast, decreasing the `Tablet time delay factor` slows down the transfer of the solid tablet (i.e. *undissolved* drug amount) for all GI compartments.

I hope this helps

André

**Reply by Carrie-nothing** [reply]
thank you so much André. you help me a lot .

best wishes to you!

Carrie
